Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -212,7 +212,7 @@ Vector<Ref<WebPageProxy>> WebProcessProxy::pages() const
 {
     auto pages = mainPages();
     for (auto& remotePage : m_remotePages) {
-        if (auto* page = remotePage.page())
+        if (RefPtr page = remotePage.page())
             pages.append(*page);
```

Comment:
`page.releaseNonNull()` to avoid ref counting churn.

Reply:
to append? Do you mean 
```
page.append(page.releaseNonNull());
```

Reply:
`pages.append(page.releaseNonNull());`

Reply:
Otherwise you're ref'ing and then deref'ing unnecessarily.